.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: 🐞 Bug report
+about: Create a report to help us improve node-fetch
+title: "[BUG] "
+labels: bug
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2. 
+3. 
+4. 
+
+**Expected behavior**
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+<!--- If fetch is behaving incorrectly, please include a link to the spec or documentation that supports that your expected behavior is correct. -->
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Your Environment**
+
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+
+ | software         | version
+| ---------------- | -------
+| node-fetch          |
```

Comment:
Nitpick: But it seems there was some effort in aligning these, should these also be aligned?

Reply:
It won't actually affect it, when its rendered I think.

Reply:
Yup, the render would be the same either way. :)